<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>banche</class>
 <widget class="QDialog" name="banche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anagrafica banca</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Inserisci banca</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>106</width>
              <height>233</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>124</width>
              <height>233</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>106</width>
              <height>233</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QGroupBox" name="groupBox">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>ID</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QLineEdit" name="id">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>IBAN</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QLineEdit" name="iban"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Nome banca</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="2">
              <widget class="QLineEdit" name="banca"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>CAP</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Citta</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1" colspan="2">
              <widget class="QLineEdit" name="indirizzo"/>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Indirizzo</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2" rowspan="2">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="cap"/>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="citta"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Lista banca</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView">
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>21</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="salva">
       <property name="toolTip">
        <string>Salva</string>
       </property>
       <property name="text">
        <string>Salva</string>
       </property>
       <property name="icon">
        <iconset resource="../fabaria.qrc">
         <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="nuovo_b">
       <property name="text">
        <string>Nuovo</string>
       </property>
       <property name="icon">
        <iconset resource="../fabaria.qrc">
         <normaloff>:/images/documentation.png</normaloff>:/images/documentation.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="elimina_b">
       <property name="toolTip">
        <string>Elimina</string>
       </property>
       <property name="text">
        <string>Elimina</string>
       </property>
       <property name="icon">
        <iconset resource="../fabaria.qrc">
         <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pdf_ex">
       <property name="text">
        <string>Esporta pdf</string>
       </property>
       <property name="icon">
        <iconset resource="../fabaria.qrc">
         <normaloff>:/images/pdf.png</normaloff>:/images/pdf.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="esci">
       <property name="toolTip">
        <string>Esci da anagrafica
banca.</string>
       </property>
       <property name="text">
        <string>Esci</string>
       </property>
       <property name="icon">
        <iconset resource="../../backup_restore_db/images.qrc">
         <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="modifica">
       <property name="text">
        <string>Modifica</string>
       </property>
       <property name="icon">
        <iconset resource="../fabaria.qrc">
         <normaloff>:/images/document-edit.png</normaloff>:/images/document-edit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>id</tabstop>
  <tabstop>iban</tabstop>
  <tabstop>banca</tabstop>
  <tabstop>cap</tabstop>
  <tabstop>citta</tabstop>
  <tabstop>indirizzo</tabstop>
  <tabstop>nuovo_b</tabstop>
  <tabstop>salva</tabstop>
  <tabstop>modifica</tabstop>
  <tabstop>elimina_b</tabstop>
  <tabstop>pdf_ex</tabstop>
  <tabstop>esci</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="../../backup_restore_db/images.qrc"/>
  <include location="../fabaria.qrc"/>
 </resources>
 <connections/>
</ui>
